faq.md

```diff
@@ -1,111 +1,67 @@
-      <li>UV spectrally resolved irradiance at surface</li>
-    </ul>
-  </div>
-  But the ESA have <a href="https://sentinel.esa.int/web/sentinel/missions/sentinel-5/data-products">more</a>.<br>
-<br></details>
+In addition to the API, you will find a few additional useful tools on our [GitHub organization page](https://github.com/emissions-api)
+- [sentinel5dl](https://github.com/emissions-api/sentinel5dl">sentinel5dl) automates downloads from the ESA archives
+- [sentinel5algorithm](https://github.com/emissions-api/sentinel5algorithms) provides several methods for working with and processing the data files
+</details>
 
 <details>
-<summary><h3> > Which products can be processed via this API?</h3></summary>
-  <div style="margin-left: 20px;">
-    Currently only carbon monoxide (CO).<br>
-  </div>
-<br></details>
+<summary>
+What is the source of the data?
+</summary>
 
-<details>
-<summary><h3> > Over which time of period are the data available?</h3></summary>
-  <div style="margin-left: 20px;">
-    The life-time of that satellite is 7.5 years. A successor satellite is planned.<br>
-  </div>
-<br></details>
+Data source is the [European Space Agency's](https://www.esa.int) satellite [Sentinel-5P](https://esa.int/Applications/Observing_the_Earth/Copernicus/Sentinel-5P).
+</details>
 
 <details>
-<summary><h3> > Which products over, what time are availabe via Emissions API?</h3></summary>
-  <div style="margin-left: 20px;">
-    We started with one product (CO) and imported from january til september 2019. But we are constantly working on importing more products over a longer period.<br>
-  </div>
-<br></details>
+<summary>
+What types of emissions are available?
+</summary>
 
-<details>
-<summary><h3> > How can I participate?</h3></summary>
-  <div style="margin-left: 20px;">
-    This is an open source project hosted on github. This means everybody can bring input and help improving the project, by e.g.:
-    <ul>
-      <li>using the API (tag the project) and create awesome use cases by visualizing the data</li>
-      <li>forking the project and working on issues</li>
-      <li>collaborate on the documentation</li>
-      <li>creating issues with new ideas</li>
-    </ul>
-    For more information about how to contribute see <a href="https://github.com/emissions-api/emissions-api/blob/master/CONTRIBUTING.rst">here</a>.<br>
-  </div>
-<br></details>
+Right now, Emissions API serves carbon monoxide measurements only.
+Several additional data products are available any may find their way into our API in the future.
+A few examples include:
 
-<details>
-<summary><h3> > How to get in contact with emissions API?</h3></summary>
-  <div style="margin-left: 20px;">
-    Via <a href="mailto:info@emissions-api.org">info@emissions-api.org</a>, <a href="https://twitter.com/emissions_api">twitter</a>, <a href="https://mastodon.social/@emissions_api">mastodon</a> or by leaving a comment e.g. on an issue.<br>
-  </div>
- <br></details>
+- Ozone (O³)
```

Comment:
These are actually all subscript instead of superscript.

Reply:
Makes sense. Fixed.